lib/test_management_service/bitbar_account_manager.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'time'
+
+module TestManagementService
+  # Manages the accounts IDs used to access BitBar
+  class BitbarAccountManager
+
+    DEFAULT_FILE_LOCATION = 'cached_accounts'
+
+    # @param max_accounts [Integer] Maximum number of BitBar accounts useable
+    # @param timeout [Integer] The maximum amount of seconds an account may be used
+    def initialize(max_accounts, timeout = 3600)
```

Comment:
An hour by default might not be enough.  We have some test runs that take over an hour on Cocoa at times.